src/frontend/staff-portal/src/app/components/staff-workbench/ticket-status/ticket-status.component.html

```diff
@@ -2,4 +2,5 @@
     <div>Request submitted date: <span>{{ dispute.submittedDate | customDate:"EEEE, MMMM ddd y"}}</span></div>
     <div>Status: <span>{{dispute.status}}</span></div>
     <div>Assigned to: <span>{{dispute.userAssignedTo ? dispute.userAssignedTo : "Unassigned"}}</span></div>
```

Comment:
I will approve -- but note that the staff can also blank out email address if the citizen is not responding to emails requesting email address verification